client/lib/signup/themes-data.js

```diff
@@ -95,6 +132,9 @@ export const themes = [
 		slug: 'lodestar',
 		repo: 'pub',
 		fallback: true,
+		description: translate(
+			'Lodestar is a trendy one-page parallax theme designed with startups and small-business ventures in mind.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Lodestar is a trendy one-page theme designed with startups and small business ventures in mind.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).